doc/docs/master/concepts/advanced-concepts/globalID.md

```diff
@@ -4,88 +4,99 @@
 Pachyderm provides users with a simple way to follow a change throughout their DAG (i.e., Traverse Provenance and Subvenance).
 
 When you commit data to Pachyderm, your new commit has an ID associated with it that you can easily check by running `pachctl list commit repo@branch`. 
-All resulting downstream commits and jobs will then share that ID (Global Identifier).
+**All resulting downstream commits and jobs will then share that ID (Global Identifier).**
 
-!!! Info "In other terms"
+!!! Info "TLDR"
     Commit and job IDs **represent a logically-related set of objects**. 
     The ID of a commit is also the ID of any commits created along with it due to provenance relationships, 
-    as well as the ID of any jobs that were started because of the creation of those commits. 
+    as well as the ID of any jobs that were triggered because of the creation of those commits. 
```

Comment:
I wasn't sure about using "trigger" here because it has a separate meaning in our product, though it's probably natural enough to not lead to any confusion